packages/kit/src/core/sync/sync.js

```diff
@@ -24,12 +33,18 @@ export function update(config) {
 	write_root(manifest_data, output);
 	write_matchers(manifest_data, output);
 	write_types(config, manifest_data);
+	write_env(config.kit, mode);
 
 	return { manifest_data };
 }
 
-/** @param {import('types').ValidatedConfig} config */
-export function all(config) {
-	init(config);
-	return update(config);
+/**
+ * Run sync.init and sync.update in series, returning the result from
+ * sync.update.
```

Comment:
I imagine this could fit on one line
```suggestion
 * Run sync.init and sync.update in series, returning the result from sync.update.
```